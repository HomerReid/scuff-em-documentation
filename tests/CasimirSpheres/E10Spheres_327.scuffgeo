OBJECT UpperSphere
	MESHFILE Sphere_327.msh
	DISPLACED 0 0 3
	MATERIAL CONST_EPS_10
ENDOBJECT

OBJECT LowerSphere
	MESHFILE Sphere_327.msh
	MATERIAL CONST_EPS_10
ENDOBJECT
