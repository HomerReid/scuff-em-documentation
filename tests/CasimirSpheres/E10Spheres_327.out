# scuff-cas3D run on supersgj1 at 03/09/16::02:34:29
# data file columns: 
#1: transform tag
#2: z-force 
#3: z-force error due to numerical Xi integration 
3.0 -3.531873e-03 1.070818e-06 
3.4 -8.898217e-04 2.518903e-07 
3.8 -2.910625e-04 7.677720e-08 
4.3 -9.052452e-05 2.402404e-08 
4.9 -2.785229e-05 7.497967e-09 
5.5 -1.014028e-05 2.749406e-09 
6.2 -3.636596e-06 1.022617e-09 
7.0 -1.309203e-06 3.823743e-10 
7.8 -5.319530e-07 1.598557e-10 
8.9 -1.792119e-07 5.605788e-11 
10.0 -6.901743e-08 2.234502e-11 
12.6 -1.054244e-08 3.819057e-12 
15.8 -1.692488e-09 7.281253e-13 
20.0 -2.534587e-10 1.481386e-13 
25.0 -4.216156e-11 3.655417e-14 
33.0 -4.535999e-12 6.780360e-15 
41.0 -9.683786e-13 2.143696e-15 
50.0 -1.614901e-13 5.613869e-16 
63.0 -2.524092e-14 1.386793e-16 
79.0 -4.093831e-15 3.464200e-17 
100 -6.142716e-16 7.158425e-18 
